apps/sage_intacct/queue.py

```
@@ -70,13 +88,13 @@ def schedule_journal_entries_creation(
         })
 
+        skip_export_count = 0
         for index, expense_group in enumerate(expense_groups):
             skip_export = validate_failing_export(is_auto_export, interval_hours, expense_group)
-
             if skip_export:
-                logger.info('Skipping export for expense group %s', expense_group.id)
-                if triggered_by == ExpenseImportSourceEnum.DIRECT_EXPORT:
-                    post_accounting_export_summary_for_skipped_exports(expense_group=expense_group, workspace_id=workspace_id)
+                skip_export_count = handle_skipped_exports(
+                    expense_groups=expense_groups, index=index, skip_export_count=skip_export_count, expense_group=expense_group, triggered_by=triggered_by
```

Pylint (3.3.7)

[convention] Line too long (155/120)

(C0301)